src/api/protectedApiClient.ts

```diff
@@ -13,13 +14,16 @@ export interface ProtectedApiClient {
   readonly getRequestStatuses: () => Promise<PersonalRequest[]>;
   readonly makePFRequest: () => Promise<void>;
   readonly deactivateAccount: () => Promise<void>;
+  readonly getContactInfo: () => Promise<ContactInfo>;
+  readonly setContactInfo: (request: ContactInfo) => Promise<void>;
```

Comment:
Why put this in redux?

Reply:
Seems like this really wouldn't need to be in redux. Not too sure though. @jackblanc thoughts?

Reply:
Doesn't necessarily have to use Redux. The alternative is local `useState` using an `AsyncRequest` object. I was not very clear at the beginning of this semester that Redux doesn't always have to be used.